packages/forms/src/UIForm/UIForm.component.test.js

```diff
@@ -331,19 +343,37 @@ describe('UIForm component', () => {
 		});
 
 		it('should call submit callback when form is valid', () => {
-			// given
 			const validProperties = {
 				...data.properties,
 				lastname: 'This has at least 10 characters',
 				firstname: 'This is required',
 			};
+			// given
```

Comment:
wat
The const just above is part of the `given`, why did you move this comment ?